src/main/Config.cpp

```diff
@@ -197,6 +197,19 @@ Config::Config() : NODE_SEED(SecretKey::random())
     FLOOD_ARB_TX_BASE_ALLOWANCE = 5;
     FLOOD_ARB_TX_DAMPING_FACTOR = 0.8;
 
+    // TODO:
+    // These flags are useful for development
+    // and it makes sense to enable pull mode by default
+    // while developing, but before shipping this code:
+    // 1. We should remove unnecessary flags
+    // 2. Flip `ENABLE_PULL_MODE` to false.
+    FLOOD_MAX_DEMAND_SIZE = 200;
```

Comment:
I thought that `FLOOD_MAX_DEMAND_SIZE` was going to be derived from `FLOOD_OP_RATE_PER_LEDGER` ?

Reply:
(and probabaly the same for `FLOOD_MAX_ADVERT_SIZE`)
note: it doesn't mean we don't have those settings, just that they're derived from others instead of being exposed in the config file

Reply:
`FLOOD_MAX_DEMAND_SIZE` is in txs, while `FLOOD_OP_RATE_PER_LEDGER` is in ops. Now we only limit the operation rate for flooding hashes (and we actually may want to revisit its value). It also seems possible to send demands with high operation counts (if peer manages to accumulate too many adverts), though not sure if that's actually an issue.

Reply:
@dmkozh  I created an issue for this: https://github.com/stellar/stellar-core/issues/3514

Reply:
> I thought that FLOOD_MAX_DEMAND_SIZE was going to be derived from FLOOD_OP_RATE_PER_LEDGER ?

That makes sense, but I'm not sure how to best do that. I'd like to set `FLOOD_MAX_DEMAND_SIZE` using `getLastMaxTxSetSizeOps` and `getExpectedLedgerCloseTime`, but Config doesn't have access to `Application`.

Reply:
I added `getMaxDemandSize` and `getMaxAdvertSize` and removed the corresponding config values.